### PR TITLE
Add single select option to RplChecklist

### PR DESCRIPTION
**Pull request by alan-cole**
**JIRA issue:** No ticket

### Changed
These are the changes required to support the Data.vic CKAN search form.

* Add single select option to form rplchecklist.

### Screenshots

Screen Shot 2019-03-20 at 9 12 18 pm
Screen Shot 2019-03-20 at 9 12 10 pm

### CODE REVIEW

**Comment by tim-yao** [comment]
Hi @dylankelly as we discussed, please merge this before your https://github.com/dpc-sdp/ripple/pull/292 merged. Cheers.

File: packages/Molecules/Form/package.json [hunk 1]
```diff
@@ -15,7 +15,6 @@
     "pikaday": "^1.7.0",
     "vue-form-generator": "^2.2.2",
     "vue-multiselect": "^2.1.3",
-    "vue-on-click-outside": "^1.0.3",
```
Comment (tim-yao) [inline packages/Molecules/Form/package.json]: Is that not required any more? @alan-cole @dylankelly ?

Comment (alan-cole) [reply packages/Molecules/Form/package.json]: That library was removed as it was causing issues with keyboard navigation. Dylan and I had discussed the possibility of including the custom implementation in a mixin - though I'm not sure what became of that.

Comment (dylankelly) [reply packages/Molecules/Form/package.json]: @alan-cole I will implement the mixin in #292

**Pull request merged**